pkg/appstate/ingress.go

```diff
@@ -165,9 +166,22 @@ func ingressGetStateFromBackend(clientset kubernetes.Interface, namespace string
 }
 
 func ingressGetStateFromExternalIP(ing *networkingv1.Ingress) types.State {
-	lbIps := loadBalancerStatusIPs(ing.Status.LoadBalancer)
+	lbIps := ingressLoadBalancerStatusIPs(ing.Status.LoadBalancer)
 	if len(lbIps) > 0 {
 		return types.StateReady
 	}
 	return types.StateUnavailable
 }
+
+func ingressLoadBalancerStatusIPs(s networkingv1.IngressLoadBalancerStatus) sets.String {
```

Comment:
had to make this change in the SDK too: https://github.com/replicatedhq/replicated-sdk/pull/60/files#r1272564106